core/desugarSwitchFuns.ml

```diff
@@ -28,19 +28,18 @@ open SourceCode
 
 let with_pos = SourceCode.WithPos.make
 
-let pattern_matching_sugar =
+let enabled =
```

Comment:
We usually have the convention that the OCaml variable is named the same as the setting itself.

Reply:
I don't have strong feelings about this. I think this convention is accidental, and I may be sole responsible for it, when I centralised most of the settings years ago as a quick fix to make all the side effects observable. With centralised settings, the names are global, and thus it makes sense to name them according to their configuration key name. With decentralised settings, some names are local, and thus we can choose names that better convey the nature of settings objects in their local context.